eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -371,27 +374,8 @@ public void testInvalidConfigJSON() {
             return null; // doAnswer doesn't require a return value
         }).when(mockHttpClient).get(anyString(), any(RequestCallback.class));
 
-        Field httpClientOverrideField = null;
-        try {
-            // Use reflection to set the httpClientOverride field
-            httpClientOverrideField = EppoClient.class.getDeclaredField("httpClientOverride");
-            httpClientOverrideField.setAccessible(true);
-            httpClientOverrideField.set(null, mockHttpClient);
-
-
-            initClient(TEST_HOST, true, true, false, DUMMY_API_KEY);
-        } catch (NoSuchFieldException | IllegalAccessException e) {
-            throw new RuntimeException(e);
-        } finally {
-            if (httpClientOverrideField != null) {
-                try {
-                    httpClientOverrideField.set(null, null);
-                } catch (IllegalAccessException e) {
-                    throw new RuntimeException(e);
-                }
-                httpClientOverrideField.setAccessible(false);
-            }
-        }
+        setHttpClientOverrideField(mockHttpClient);
```

Comment:
Cleaned up the modification of the private static override variables into a little helper function 🧹